ansible/playbooks/roles/explorer_be/files/firestore-service-account-key.json

```diff
@@ -0,0 +1,125 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
question: is this some kind of encryption on the key json file?

Reply:
yeah. it is encrypted